template-sample-app/template/src/components/views/Kpi.js

```diff
@@ -1,6 +1,6 @@
 import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { setBottomSheetOpen, setError } from 'config/appSlice';
+import { setBottomSheetOpen, setError } from 'store/appSlice';
```

Comment:
The same, only includes here the required stuff

Reply:
This is required by the new slice hygen generator, as they Arana explained us in a meet. I'm ok with that refactor, we just need to check doc is well updated on this

Reply:
Done